.gitignore

```diff
@@ -4,3 +4,4 @@
 __pycache__/
 data/
 */__pycache__/
+toto
```

Comment:
to be removed

Reply:
toto is the output of model history. I think it not necessary to keep it here...

Reply:
I got it removed. Just wonder why? :)

Reply:
nan mais toto c'est pas bon comme nom!